### PR TITLE
Fix post path comment

### PR DESCRIPTION
**Pull request by mottox2**
I thought that the post will be put /post/{slug} , but it is put /${slug}.
so I changed the comment.

### CODE REVIEW

File: examples/using-wordpress/gatsby-node.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ exports.createPages = ({ graphql, boundActionCreators }) => {
           // The Post ID is prefixed with 'POST_'
           _.each(result.data.allWordpressPost.edges, edge => {
             createPage({
-              path: edge.node.slug,
+              path: `/${edge.node.slug}/`,
```
Comment (pieh) [inline examples/using-wordpress/gatsby-node.js]: This change breaks prefetching, please revert this and leave just routes comment change (which is valid)

Comment (mottox2) [reply examples/using-wordpress/gatsby-node.js]: OK. I fixed.

**Comment by pieh** [comment]
Thanks!

**Comment by jlengstorf** [comment]
Hiya @mottox2! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs

**Pull request merged**